src/elements/call-to-action.html

```diff
@@ -9,45 +9,47 @@
 
   <template>
 
-    <style include="shared-styles">
-      :host {
-        display: block;
-        background: var(--default-primary-color);
-        color: var(--text-primary-color);
-      }
-
-      .container {
-        @apply(--layout-vertical);
-        @apply(--layout-center-center);
-      }
-
-      .heading {
-        @apply(--layout-flex-auto);
-        @apply(--layout-self-center);
-        margin-bottom: 0 !important;
-      }
-
-      @media (max-width: 599px) {
-        .open-dialog-button {
-          margin-top: 10px;
+    {% if callToAction %}
```

Comment:
@sophieH29 as for featured videos there are two different components which can be independent:

- videos themselves (which are stored in the firebase, as I got it)
- button link to youtube playlist, which is set in the configuration

However they both are handled with the same component, which makes it difficult to handle them the way you requested.